src/goose/api/batch.clj

```diff
@@ -0,0 +1,27 @@
+(ns goose.api.batch
+  (:require [goose.broker :as b]
+            [goose.client]))
+
+(defn- status-from-counts
+  [{:keys [successful dead]}
+   total]
+  (if (= (+ successful dead) total)
+    "complete"
```

Comment:
suggestion: lets have a high level definition for status of batch (e.g in a `def`) ?

Reply:
Moved the strings to a `def`